e2e-tests/IIoTPlatform-E2E-Tests/Orchestrated/T_TwinBrowseOrchestratedTestTheory.cs

```diff
@@ -0,0 +1,175 @@
+﻿// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+namespace IIoTPlatform_E2E_Tests.Orchestrated {
+    using IIoTPlatform_E2E_Tests.TestExtensions;
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading;
+    using Xunit;
+    using Xunit.Abstractions;
+
+    [TestCaseOrderer(TestCaseOrderer.FullName, TestConstants.TestAssemblyName)]
+    [Collection("IIoT Multiple Nodes Test Collection")]
+    [Trait(TestConstants.TraitConstants.PublisherModeTraitName, TestConstants.TraitConstants.PublisherModeOrchestratedTraitValue)]
+    public class T_TwinBrowseOrchestratedTestTheory {
+        private readonly IIoTMultipleNodesTestContext _context;
+
+        public T_TwinBrowseOrchestratedTestTheory(IIoTMultipleNodesTestContext context, ITestOutputHelper output) {
+            _context = context ?? throw new ArgumentNullException(nameof(context));
+            _context.OutputHelper = output ?? throw new ArgumentNullException(nameof(output));
+        }
+
+        [Fact, PriorityOrder(0)]
+        public void A1_SwitchToOrchestratedMode() {
+            TestHelper.SwitchToOrchestratedModeAsync(_context).GetAwaiter().GetResult();
+        }
+
+        [Fact, PriorityOrder(1)]
+        public void A2_RegisterOPCServer_And_ActivateEndpoint() {
+            var cts = new CancellationTokenSource(TestConstants.MaxTestTimeoutMilliseconds);
+
+            // We will wait for microservices of IIoT platform to be healthy and modules to be deployed.
+            TestHelper.WaitForServicesAsync(_context, cts.Token).GetAwaiter().GetResult();
+            _context.RegistryHelper.WaitForIIoTModulesConnectedAsync(_context.DeviceConfig.DeviceId, cts.Token).GetAwaiter().GetResult();
+
+            var simulatedOpcServer = TestHelper.GetSimulatedPublishedNodesConfigurationAsync(_context, cts.Token).GetAwaiter().GetResult();
```

Comment:
confusing naming: are we getting the simulated server or the simulated published nodes here?